RocketApp/Domain/Sources/RocketsClient/RocketsErrorManager/RocketsClientAsyncError.swift

```diff
@@ -12,7 +12,7 @@ public enum RocketsClientAsyncError: Error, Equatable, CustomDebugStringConverti
   public var description: String {
```

Comment:
Tenhle error ma conformovat `ErrorReporting`? Nebo jako vsechny errory musi conformovat bud `ErrorReporting` nebo `CombineErrorReporting`.

Reply:
Jinak ja bych asi nechal jen jeden error, kterej conformuje oboje. Tohle je asi zbytecny mit 2 ruzny errory ne?

Reply:
Konformance pridana. Kazdy z erroru by mohl mit vlastni identitu na zaklade pouzityho klienta. Mozna by se to mohlo sjednotit. Jeste to musim promyslet, zatim bych nechal rozdeleny.

Reply:
Nevim co myslis tou identitou ale nema smysl rozhodne mit vic erroru pro tu stejnou vec. Tak to proste jen sjednot ne? Dyt akorat jeden error smazes a pridas conformance pro ten druhej...

Reply:
Nebo to nech takhle no zatim...